Formula/libpostal.rb

```diff
@@ -0,0 +1,44 @@
+class Libpostal < Formula
+  desc "C library for parsing or normalizing addresses"
+  homepage "https://github.com/openvenues/libpostal"
+  url "https://github.com/openvenues/libpostal/archive/v0.3.1.tar.gz"
+  head "https://github.com/openvenues/libpostal.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "snappy" => :run
```

Comment:
Don't need to specify `:run` for most things. Only the ones the audit suggests should be build-only, or special dependencies (which none of these are). Just `depends_on "snappy"` is fine here.